examples/docker-compose/README.md

```diff
@@ -0,0 +1,60 @@
+# Docker Compose + Fluvio
+
+You can run Fluvio Clusters in a Docker Compose setup, this could be useful for
+local development and POC development.
+
+In order to run a Fluvio Cluster through Docker, you will need to run Fluvio
+components separately, we can use Docker Compose `service`s to achieve this.
+
+## Services
+
+- `sc: Streaming Controller`
+- `sc-setup: Post-Initialization Commands`
+- `spu: Streaming Processing Unit`
+
+> To learn more about Fluvio architecture, please refer to [Fluvio Documentation][1]
+
+## Running Locally
+
+Clone this repo using `git clone https://github.com/infinyon/fluvio.git` and
```

Comment:
Might need to add `mkdir metadata data`.  This makes it so starting back up retains both the metadata state and the data of the compose cluster.

Reply:
Docker takes care of creating them! I prefixed with `fluvio-` in the user side to avoid conflicts with local dirs!